Src/pangen3.c

```diff
@@ -296,7 +296,10 @@ comwork(FILE *fd, Lextok *now, int m)
 			&&  now->sym->mtype_name)
 			{	s = now->sym->mtype_name->name;
 			}
-			sr_mesg(fd, now->val, now->ismtyp, s);
+			if (now->constValKind == VALUE_FLOAT)
+				sr_mesg_f(fd, getFloatTokenValue(now), now->ismtyp, s);
+			else
+				sr_mesg(fd, now->val, now->ismtyp, s);	// TODO PG - inside there is comparison of now->val to 1 and printf as int...
```

Comment:
I don't understand the comment in the context of the current version of the code.

Reply:
ok - the comment was left as TODO, why it should be descriptive, why I needed two versions of sr_mesg function...

Reply:
will be fixed